components/integrations/installed_command.test.tsx

```diff
@@ -40,13 +42,13 @@ describe('components/integrations/InstalledCommand', () => {
     });
 
     test('should match snapshot, not autocomplete, no display_name/description/auto_complete_hint', () => {
-        const minCommand = {
+        const minCommand = TestHelper.getCommandMock({
             id: 'r5tpgt4iepf45jt768jz84djic',
             trigger: 'trigger',
             auto_complete: false,
             token: 'testToken',
-            create_at: '1499722850203',
-        };
+            create_at: 1499722850203,
```

Comment:
The create_at changed from string to int, is that intended?

Reply:
@javaguirre yes, as `type` of create_at is number, at least that’s what getting inferred

Reply:
In that case, I don't have anything else to add @VishakhaPoonia

I think there are conflicts, so if we resolved those I'm good with it. :-)

Thank you!